robolectric/pom.xml

```diff
@@ -131,6 +131,9 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
+        <configuration>
```

Comment:
It's not necessary, without the enlarged test memory space, robolectric sometimes throw OutofMemoryError when running install-robolectric, then I have to re-run the script and wait another 5 minutes.